modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/helper/AddFormInstanceRecordMVCCommandHelperTest.java

```diff
@@ -371,6 +391,18 @@ protected void setUpAddRecordMVCCommandHelper() throws Exception {
 			_addRecordMVCCommandHelper, _portal
 		);
 
+		FFSubmissionsSettingsConfigurationActivator
+			ffSubmissionsSettingsConfigurationActivator =
+				_mockFFSubmissionsSettingsConfigurationActivator();
+
+		field(
+			AddFormInstanceRecordMVCCommandHelper.class,
+			"_ffSubmissionsSettingsConfigurationActivator"
+		).set(
+			_addRecordMVCCommandHelper,
+			ffSubmissionsSettingsConfigurationActivator
+		);
+
```

Comment:
@rodrigopaulino shouldn’t we put this before `AddFormInstanceRecordMVCCommandHelper.class, "_portal"`?
I'm asking because in `AddFormInstanceRecordMVCCommandHelper` the `_ffSubmissionsSettingsConfigurationActivator` variable is declared before.

Reply:
We should. Also my bad.